packages/chakra-ui-core/src/Progress/Progress.stories.js

```diff
@@ -1,45 +1,35 @@
 import { storiesOf } from '@storybook/vue'
-import { CSSReset, Progress as KProgress, Stack } from '..'
+import { CSSReset, Progress as KProgress, Stack, Box } from '..'
 
 storiesOf('UI | Progress', module)
-  .add('Default Progress', () => ({
-    components: { CSSReset, KProgress },
+  .addDecorator(story => ({
+    components: { CSSReset, Box, story: story() },
     template: `
-      <div>
-        <CSSReset />
-        <KProgress :value="80" />
-      </div>
-    `
+      <Box w="full" maxWidth="400px" mx="auto" mt="8" p="3">
+        <CSSReset></CSSReset>
+        <story></story>
```

Comment:
> Interesting. So we needed to add decorate before the story to fix this? Do you happen to know why? 😅


@codebender828 See? we can use as a wrapper. Each story renders inside `story` component. Like a `slot`. So we don't need to add `<CSSReset>` to each story.